src/Markdown/Inline.elm

```diff
@@ -0,0 +1,98 @@
+module Markdown.Inline exposing
+    ( Inline(..)
+    , extractText
+    )
+
+{-| Inline rendering and helpers.
+
+
+# Model
+
+@docs Inline
+
+
+# Helpers
+
+@docs extractText
+
+-}
+
+import HtmlParser
+import Markdown.Block as Block
+
+
+{-| The inline type.
+
+  - **Text** | _Text_
+  - **HardLineBreak**
+  - **CodeInline** | _Code_
+  - **Link** | _Url_ | _Maybe Title_ | _Inlines_
+  - **Image** | _Source_ | _Maybe Title_ | _Inlines_
+  - **HtmlInline** | _Tag_ | _List ( Attribute, Maybe Value )_ | _Inlines_
+  - **Emphasis** | _Delimiter Length_ | _Inlines_
+
+-}
+type Inline htmlValue
```

Comment:
I can't tell what the difference is between this type and the one defined in `Markdown.Block`. Are they used for different purposes? I see that `Emphasis` has an `Int` param here but not there.

Reply:
This module was copied from Pablo's code. In the context of this codebase, a better name for it would be RawInline. It isn't the final `Markdown.Block.Inline` type, but rather an intermediary type that contains Html Nodes without the children being parsed.

Reply:
Could even make sense for this to live in the `RawBlock` module.

Reply:
Ah I see that now with the parameter to `HtmlInline` being the parser node. Makes sense!

Reply:
Might be an opportunity for future refactoring to try moving it into `RawBlock` or renaming it, I'll keep that in mind.

Thanks for taking the time to look at all the code!